pandas/tests/io/json/test_readlines.py

```diff
@@ -13,49 +14,64 @@
 
 from pandas.io.json._json import JsonReader
 
+xfail_json_engine_pyarrow = pytest.mark.usefixtures("json_engine_pyarrow_xfail")
+
 
 @pytest.fixture
 def lines_json_df():
     df = DataFrame({"A": [1, 2, 3], "B": [4, 5, 6]})
     return df.to_json(lines=True, orient="records")
 
 
-def test_read_jsonl():
+@xfail_json_engine_pyarrow
```

Comment:
Hmm what is the xfail here for?

Reply:
The reason for this xfail is that currently, `pyarrow.json.read_json` only supports a path input, not a JSON string. Below is from its documentation:

> input_file: path or file-like object. The location of JSON data. Currently, only the line-delimited JSON format is supported.